Source/Core/VideoCommon/VertexLoaderBase.cpp

```diff
@@ -131,12 +131,13 @@ class VertexLoaderTester : public VertexLoaderBase
 	: VertexLoaderBase(vtx_desc, vtx_attr), a(_a), b(_b)
 	{
 		m_initialized = a && b && a->IsInitialized() && b->IsInitialized();
-		bool can_test = a->m_VertexSize == b->m_VertexSize &&
-		                a->m_native_components == b->m_native_components &&
-		                a->m_native_vtx_decl.stride == b->m_native_vtx_decl.stride;
 
 		if (m_initialized)
 		{
+			bool can_test = a->m_VertexSize == b->m_VertexSize &&
```

Comment:
What is it you mean by directly storing it. At the end of this if statement it had

```cpp
m_initialized &= can_test;
```

which I've changed to

```cpp
m_initialized = can_test;
```